aws/services/waf/resource.ftl

```diff
@@ -351,6 +360,20 @@
                     version=version /]
             [/#if]
             [#switch version]
+
+                [#switch rule.Engine ]
```

Comment:
Not sure this will work as the code isn't inside a case block for the switch based on version?

Reply:
Had it locally but didn't push.